Modules/posixmodule.c

```diff
@@ -4471,9 +4479,18 @@ os__path_splitroot_impl(PyObject *module, path_t *path)
         *p = L'\\';
     }
 
+#ifdef MS_WINDOWS_GAMES
+    /* this does not handle persistent local storage */
+    ret = E_FAIL;
+    if ((wcsncmp(buffer, L"G:\\", 3) == 0) || (wcsncmp(buffer, L"D:\\", 3) == 0) || (wcsncmp(buffer, L"T:\\", 3) == 0)) {
+        end = buffer + 3;
+        ret = S_OK;
+    }
```

Comment:
This is an accelerator function that has never actually been used in `ntpath`. It will probably get updated to match how `ntpath.splitroot()` works and used, maybe in 3.12 or 3.13. For now, the definition of `os__path_splitroot_impl` should simply be skipped on Xbox.